### ISSUE TITLE
G207: Image missing  "Orange icon on white background"

### ISSUE THREAD

**Issue by jake-abma** [w3c/wcag#645]
at technique "Ensuring that a contrast ratio of 3:1 is provided for icons": https://www.w3.org/WAI/WCAG21/Techniques/general/G207 a image is missing => "Orange icon on white background"

**Comment by awkawk** [comment]
Fixed: a455f96
